modules/network-observability-flowcollector-view.adoc

```diff
@@ -50,14 +50,15 @@ spec:
       conversationHeartbeatInterval: 30s
   loki:                     <4>
     mode: LokiStack         <5>
+    readTimeout: 3m0s       <6>
```

Comment:
I would recommend to keep the `30s` default in the example here.

Reply:
+ consider adding the extra ones:
```suggestion
    readTimeout: 30s       <6>
    writeTimeout: 10s
    writeBatchWait: 1s
    writeBatchSize: 102400
    advanced:
      writeMinBackoff: 1s
      writeMaxBackoff: 5s
      writeMaxRetries: 2
      staticLabels: 
        app: netobserv-flowcollector
```

https://github.com/netobserv/network-observability-operator/blob/d9eb3694e04237c40e32ec679cc5bdb7d894b9d7/apis/flowcollector/v1beta2/flowcollector_types.go#L741-L763

Reply:
Ok I will keep it to 30s. Idk about adding callouts for all these, but I can see adding them for visibility. Do you think they should all have code callouts? I know generally we want to de-emphasize Loki in the docs, but I guess I am inclined to add the `readTimeout` because someone asked for it in a bug.

Reply:
Indeed, that's why the only loki option displayed in the doc here was the `mode`. I'm not especially in favor of adding too much informations here honnestly. 
Keeping the first install example simple makes things easy for the user to understand. We could even get rid of some sections such as `processor.advanced`, `resources`, `logLevel` etc. 

I would prefer having dedicated pages per component for extra explanations or just rely on the Custom Resource Definition as before.

@jotak any thoughts ?

Reply:
Okay, thanks for the discussion around this. I will not document these Loki options here and will instead link to the API reference. As Joel points out, its not visible at all in the Configuring section, and maybe just linking there is sufficient. There's many configs in the CRD that we don't bring directly to this sample, and these Loki configs just don't seem to be **_the_** **_thing_** to bring here.